### DISCUSSION TITLE
more advanced sample ?

### DISCUSSION THREAD

**Discussion by Masoud-Ebrahimi** [HashLoad/horse#397] [General]
Hi ,
I founded HORSE project and it was really usefull.
The samples are really simple but I need advanced samples. for example for using ssl , authentication , security in api , stream data transfer and etc.
May I have more useful and advanced samples  ?

**Comment by Dzyszla** [comment]
Stream - look at https://github.com/HashLoad/horse-octet-stream
SSL - look f.ex. into https://github.com/HashLoad/horse/releases/tag/3.1.0
Authentication - look at https://github.com/HashLoad/horse-basic-auth or https://github.com/HashLoad/horse-jwt or https://github.com/andre-djsystem/horse-bearer-auth
